.classpath

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If I'm not mistaken, this file is automatically created by `eclipse`, so you don't need it here on repository.

FYI To the repository, you should add project files, that are responsible for project realization. For example: if you are using some library, then you don't need to add it to the repository, as the library will be the same for all, if you will use the same library version and download from the same repository.

So the main idea to save on repository files and configurations. For git you can find and use `.gitignore` file to skip adding not necessary files to the project with examples ;)